core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -541,7 +541,7 @@ public void configure() throws ConfigurationException {
 			super.configure();
 			if (StringUtils.isEmpty(getName())) {
 				if (getListener()!=null) {
-					setName(ClassUtils.nameOf(getListener()));
+					setName(Misc.concatStrings(ClassUtils.nameOf(getListener()), " ", getListener().getName()));
```

Comment:
What does this do? This overwrites the name of the receiver? do we wan tthis? what if it's already set?

Reply:
If the receiver does not have a name and has a listener the name of the receiver will become the class name of listener + listenerName(in case it is not empty)